nixos/modules/flyingcircus/roles/statshost.nix

```diff
@@ -5,24 +5,70 @@
 with lib;
 
 let
-  cfg = config.flyingcircus.roles.statshost;
-  cfg_proxy = config.flyingcircus.roles.statshostproxy;
-
-  # can be replaced with pkgs.influxdb once InfluxDB 0.11 is present in upstream
-  # nixpkgs.
-  influxdb = (pkgs.callPackage ../packages/influxdb.nix { }).bin //
-    { outputs = [ "bin" ]; };
-
-  statshost_service = lib.findFirst
+  fclib = import ../lib;
+
+  cfgStatsGlobal = config.flyingcircus.roles.statshost;
+  cfgStatsRG = config.flyingcircus.roles.statshost-master;
```

Comment:
What's the wording here? "master" -> rg local? "no suffix" (which existed before) is our stats.flyingcircus.io?

Could you explain the architecture in a little more detail?